Islandora-Devops/claw-playbook
# Update modules: Issue 1052

**GitHub Issue**:  /Islandora-CLAW/CLAW/issues/1052

# What does this Pull Request do?

Bumps composer versions for devel, rdfui, restui, and facets modules.

# What's new?

Changes
  - "drupal/devel:^1.0@beta" → ^2.0
  - "drupal/rdfui:1.x-dev" → ^1.0-beta1
  - "drupal/restui:^1.13" → ^1.16
  - "drupal/facets:1.x-dev" → ^1.3

# How should this be tested?

- checkout a fresh claw-playbook
- apply the PR
- vagrant up should complete as usual
- make sure nothing broke.

# Additional Notes:
Related PR: /Islandora-CLAW/islandora/pull/122

You can test them both at once by applying this PR first and then updating the drupal.yml according to the full-testing instructions on the other PR.

# Interested parties
@Islandora-CLAW/committers, esp @dannylamb
